packages/app/src/components/DynamicRoot/DynamicRoot.tsx

```diff
@@ -35,8 +35,8 @@ export type StaticPlugins = Record<
   {
     plugin: BackstagePlugin;
     module:
-      | React.ComponentType<{}>
-      | { [importName: string]: React.ComponentType<{}> };
+      | React.ComponentType<any>
```

Comment:
What's this change for?

Reply:
I had some type issues, but I reverted that change now. :+1: